app/src/main/java/com/example/solar_alarm/Alarm.java

```diff
@@ -4,21 +4,24 @@
 
 import androidx.annotation.RequiresApi;
 
+import java.io.Serializable;
 import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 
-public class Alarm
+public class Alarm implements Serializable
 {
-    private LocalTime AlarmTime;
+    private static ArrayList<Alarm> contacts = new ArrayList<Alarm>(1);
+    private String AlarmTime;
     private String    AlarmName;
 
-    public Alarm(LocalTime alarmTime, String alarmName)
+    public Alarm(String alarmTime, String alarmName)
     {
         AlarmTime = alarmTime;
         AlarmName = alarmName;
     }
 
-    public LocalTime GetAlarmTime()
+    public String GetAlarmTime()
```

Comment:
Make this into a formatter.

https://www.geeksforgeeks.org/localtime-format-method-in-java-with-examples/